src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/set_brgemm_copy_b_buffers_shape.cpp

```diff
@@ -28,11 +29,10 @@ bool ov::intel_cpu::pass::SetBrgemmCopyBBuffersShape::run(snippets::lowered::Lin
         const auto& expr = *expr_it;
         if (auto copy_b = ov::as_type_ptr<ov::intel_cpu::BrgemmCopyB>(expr->get_node())) {
             const auto buffer = get_buffer_from_output(expr, 0);
-            const auto buffer_shape = copy_b->get_repacking_buffer_shape();
-            buffer->set_allocation_size(ov::shape_size(buffer_shape));
+            buffer->set_allocation_size(ov::intel_cpu::jit_brgemm_copy_b_emitter::get_repacking_buffer_size(expr));
             if (with_compensations(copy_b->get_type())) {
                 const auto compensations_buffer = get_buffer_from_output(expr, 1);
-                compensations_buffer->set_allocation_size(ov::shape_size(copy_b->get_compensations_buffer_shape()));
+                compensations_buffer->set_allocation_size(ov::intel_cpu::jit_brgemm_copy_b_emitter::get_compensations_buffer_size(expr));
```

Comment:
Same suggestion as for other static utility methods in `jit_brgemm_copy_b_emitter`:
Let's move them all to brgemm_utils. If necessary, we can place then in a separate namespace to distinguish between brgemm and brgemm_copy_b - related functions.